src/main/java/com/google/sps/servlets/AddAvailabilityServlet.java

```diff
@@ -19,37 +19,24 @@
 import com.google.sps.data.TutorSession;
 import com.google.sps.data.SampleData;
 import com.google.sps.utilities.AvailabilityDatastoreService;
-import com.google.sps.utilities.RealAvailabilityDatastore;
-import com.google.sps.utilities.MockAvailabilityDatastore;
 import com.google.gson.Gson;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Calendar;
+import java.util.Optional;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/addAvailability")
+@WebServlet("/add-availability")
```

Comment:
Is hyphenation the default way to name servlets?

Reply:
The DeleteAvailabilityServlet used hyphenation, so I changed this one too. I'm not sure if there is a default way.

Reply:
yep, that sounds good. I'll start using hyphenation too:)